modules/samples/artifacts/CDCSample/CDCWithPollingMode.siddhi

```diff
@@ -0,0 +1,65 @@
+/*
+Purpose:
+    This sample demonstrates how to use polling mode of cdc source.
+    A MySQL table's INSERTS are captured.
+
+Pre-requesites:
+    1) Ensure that MySQL is installed on your computer.
+    2) Add the MySQL JDBC driver into {WSO2_SP_HOME}/lib as follows:
+            2.1) Download the JDBC driver from: https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.45.tar.gz
+            2.2) Unzip the archive.
+            2.3) Copy mysql-connector-java-5.1.45-bin.jar to {WSO2_SP_Home}/lib directory.
+    3) Enable state persistence in siddhi apps.
```

Comment:
It will work without state persistence. But in order to capture changes which occurred when the cdc poller is down, have to enable state.persistence. otherwise it will fail to capture those changes.

Reply:
We can mention that in the documentation and that is not needed just to check the sample right? Shall we put it as optional and give the concerns

Reply:
fixed with commit: 9222e36